README.md

```diff
@@ -1,142 +1,170 @@
-  
-  ```bash
-  pip uninstall mhkit
-  cd MHKiT-Python
-  pip install -e .
-  ```
-   **NOTE:** After building the documentation (see below) be sure to reinstall your MHKiT-Python distribution from pypi (e.g. `pip install mhkit`) or from your fork of the repository as an editable package (e.g. `pip install -e /path/to/my/fork/of/MHKiT-Python`)
 
-### Step 2. Download/Install [Sphinx](http://www.sphinx-doc.org/en/stable/index.html) package and extensions
-  - [Sphinx]((http://www.sphinx-doc.org/en/master/usage/installation.html)) is a python package used to create the MHKiT documentation.
+- To update the documentation a user will need to "fork" this repository. This simply means that you are creating your own copy of this repository that you can edit.
+- Once forked you can "clone" (this will download your fork to your machine) the repository to your local machine using the command line.
+- Using the command line navigate to the file system location you would like to keep the documentation repository clone the documentation. Replacing the username and repository name to the correct values for you. This link can be automatically generated on your fork by copying the link visible after clicking clone on your repository page. The repository name will be assumed to be MHKiT in the remaining commands.
+
+```bash
+  git clone https://github.com/USERNAME/REPOSITORYNAME.git
+```
+
+- Once cloned use the command line to change directories into the folder created by the clone to update the MHKiT-Python and MHKiT-MATLAB [submodules].
+
+```bash
+   cd MHKiT
+   git submodule init
+   git submodule update --remote
+```
+
+The `MHKiT-Python` and `MHKiT-MATLAB` folders should now contain source code.
+
+To build the Python API documentation for submitting a pull request you will want to use the submodule MHKiT-Python included here. To do so first uninstall any MHKiT you may have and then use the python package manager to install this MHKiT submodule.
 
-  - Install Sphinx from the command line
-   **NOTE:** You may need to add PROXY settings ([see info here](https://cinhtau.net/2018/04/16/python-proxy-windows/))
-    
-    ```pip install -U Sphinx```
-  
-    - Sphinx --> Version: 8.1.3
+```bash
+pip uninstall mhkit
+cd MHKiT-Python
+pip install -e .
+```
 
-  - Use the command line to install the needed Sphinx submodules (BibTex, MATLAB theme, rtd theme, NB Sphinx, and google analytics) and additional dependencies
+**NOTE:** After building the documentation (see below) be sure to reinstall your MHKiT-Python distribution from pypi (e.g. `pip install mhkit`) or from your fork of the repository as an editable package (e.g. `pip install -e /path/to/my/fork/of/MHKiT-Python`)
 
-     ```pip install -U sphinxcontrib-bibtex sphinxcontrib-matlabdomain sphinx_rtd_theme nbsphinx sphinxcontrib-googleanalytics ipython```
```

Comment:
Since Step 2 creates a separate environment for the documentation, we can remove this step 
```suggestion

Reply:
Good catch. I moved this to be a part of the manual install option below. 
Resolved in bb4aa9c51a43d310b07475e6e586def9a859983a